manifest_test.go

```diff
@@ -25,7 +25,7 @@ func TestReadManifest(t *testing.T) {
 		t.Fatalf("Should have read Manifest correctly, but got err %q", err)
 	}
 
-	c, _ := gps.NewSemverConstraint(">=0.12.0, <1.0.0")
+	c, _ := gps.NewSemverConstraint("^0.12.0")
```

Comment:
Looking at the repo's readme, I believe those are equal. The change may not be necessary if the testdata is not updated, I'll test this out locally.